docs/bundles/SyliusFixturesBundle/builtin_listeners.rst

```diff
@@ -0,0 +1,80 @@
+Built-in listeners
+==================
+
+**SyliusFixturesBundle** comes bundled with a few useful listeners.
+
+Logger (``logger``)
+-------------------
+
+Provides output during running ``sylius:fixtures:load`` command.
+
+.. code-block:: bash
+
+    # Without logger
+
+    $ app/console sylius:fixtures:load my_suite
+    $ _
+
+    # With logger
+
+    $ app/console sylius:fixtures:load my_suite
+    Running suite "my_suite"...
+    Running fixture "country"...
+    Running fixture "locale"...
+    Running fixture "currency"...
+    $ _
+
+Has no configuration options, can be enabled by:
```

Comment:
`The Logger does not have any configuration options. It can be enabled in such a way:`